INSTALL.md

```diff
@@ -7,58 +7,45 @@ that you can download instantly and get started with fast](https://www.mapbox.co
 Still with us? These are the instructions you'll need to build Mapbox GL Native
 from source on a variety of platforms and set up a development environment.
 
-Your journey will start with getting the source code, then installing the
-dependencies, and then setting up a development environment, which varies
-depending on your operating system and what platform you want to develop for.
+Your journey will start with installing dependencies, then getting the source code, and
+then setting up a development environment, which varies depending on your
+operating system and what platform you want to develop for.
 
-## 1: Getting the source
+## 1: Installing dependencies
 
-Clone the git repository:
+### macOS
 
-    git clone https://github.com/mapbox/mapbox-gl-native.git
-    cd mapbox-gl-native
+ 1. Install [Xcode](https://developer.apple.com/xcode/)
+ 2. Install [Homebrew](http://brew.sh)
+ 3. Install [Node.js](https://nodejs.org/), [ruby](https://www.ruby-lang.org/), [CMake](https://cmake.org/), and [ccache](https://ccache.samba.org) (`brew install nodejs ruby cmake ccache`)
```

Comment:
macOS comes with Ruby 2.0.x, which, while ancient, should be sufficient for our purposes — was that not the case for you?

Reply:
Installing gems using the default ruby installation requires `sudo`. We have the choice between

 - suggesting users run `brew install ruby` 
 - suggesting users run `sudo gem install xcpretty`
 - doing neither, sending beginners on a StackOverflow quest, after which they'll probably end up running `brew install ruby`

Reply:
Ah, true — `sudo gem install` is best avoided. 👍